common/build-style/void-cross.sh

```diff
@@ -0,0 +1,421 @@
+	CC="${tgt}-gcc" CXX="${tgt}-g++" CPP="${tgt}-cpp" LD="${tgt}-ld" \
+	AS="${tgt}-as" NM="${tgt}-nm" CFLAGS="-pipe" CXXFLAGS="" CPPFLAGS="" \
+	LDFLAGS="" \
+	../glibc-${cross_glibc_version}/configure \
+		--prefix=/usr \
+		--host=${tgt} \
+		--with-headers=${wrksrc}/build_root/usr/${tgt}/usr/include \
+		--config-cache \
+		--enable-obsolete-rpc \
+		--enable-obsolete-nsl \
+		--enable-kernel=2.6.27
+
+	make -k install-headers cross_compiling=yes \
+		install_root=${wrksrc}/build_root/usr/${tgt}
+
+	touch ${wrksrc}/.glibc_headers_done
+}
+
+_build_glibc() {
+	[ -f .glibc_build_done ] && return 0
+
+	local tgt=$cross_triplet
+
+	msg_normal "Building glibc for ${tgt}\n"
+
+	mkdir -p glibc_build
+	cd glibc_build
+
+	echo "slibdir=/usr/lib${cross_wordsize}" > configparms
+
+	echo "libc_cv_forced_unwind=yes" > config.cache
+	echo "libc_cv_c_cleanup=yes" >> config.cache
+
+	CC="${tgt}-gcc" CXX="${tgt}-g++" CPP="${tgt}-cpp" LD="${tgt}-ld" \
+	AS="${tgt}-as" NM="${tgt}-nm" CFLAGS="-pipe ${cross_glibc_cflags}" \
+	CXXFLAGS="-pipe ${cross_glibc_cflags}" CPPFLAGS="${cross_glibc_cflags}" \
+	LDFLAGS="${cross_glibc_ldflags}" \
+	../glibc-${cross_glibc_version}/configure \
+		--prefix=/usr \
+		--libdir=/usr/lib${cross_wordsize} \
+		--libexecdir=/usr/libexec \
+		--host=${tgt} \
+		--with-headers=${wrksrc}/build_root/usr/${tgt}/usr/include \
+		--config-cache \
+		--enable-obsolete-rpc \
+		--enable-obsolete-nsl \
+		--disable-profile \
+		--disable-werror \
+		--enable-kernel=2.6.27 \
+		${cross_glibc_configure_args}
+
+	make ${makejobs}
+	make install_root=${wrksrc}/build_root/usr/${tgt} install
+
+	touch ${wrksrc}/.glibc_build_done
+}
+
+_build_gcc() {
+	[ -f .gcc_build_done ] && return 0
+
+	msg_normal "Building gcc for ${cross_triplet}\n"
+
+	mkdir -p gcc_build
+	cd gcc_build
+
+	../gcc-${cross_gcc_version}/configure \
+		--prefix=/usr \
+		--sbindir=/usr/bin \
+		--libdir=/usr/lib \
+		--libexecdir=/usr/lib \
+		--target=${cross_triplet} \
+		--with-sysroot=/usr/${cross_triplet} \
+		--with-build-sysroot=${wrksrc}/build_root/usr/${cross_triplet} \
+		--enable-languages=c,c++,fortran,objc,obj-c++,ada,go,lto \
+		--disable-nls \
+		--disable-multilib \
+		--disable-sjlj-exceptions \
+		--disable-libquadmath \
+		--disable-libmudflap \
+		--disable-libssp \
+		--disable-libitm \
+		--disable-libvtv \
+		--disable-libsanitizer \
+		--disable-libcilkrts \
+		--disable-libstdcxx-pch \
+		--enable-shared \
+		--enable-threads=posix \
+		--enable-long-longx \
+		--enable-linker-build-id \
+		--enable-gnu-unique-object \
+		--enable-libada \
+		--enable-lto \
+		--enable-libstdcxx-time \
+		--enable-default-pie \
+		--enable-default-ssp \
+		--with-gnu-ld \
+		--with-gnu-as \
+		--with-linker-hash-style=gnu \
+		${configure_args}
+
+	make ${makejobs}
+
+	touch ${wrksrc}/.gcc_build_done
+}
+
+do_build() {
+	# Verify toolchain versions
+
+	[ -z "$cross_binutils_version" ] && msg_error "Unknown binutils version\n"
```

Comment:
Perhaps print the expected environment variable (cross_binutils_version) in the error message as a reminder.